forge-gui/res/cardsfolder/upcoming/aspiring_champion.txt

```diff
@@ -0,0 +1,13 @@
+Name:Aspiring Champion
+ManaCost:3 R
+Types:Creature Astartes Warrior
+PT:3/3
+K:Menace
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigDig | TriggerDescription$ Ruinous Ascension — When CARDNAME deals combat damage to a player, sacrifice it. If you do, reveal cards from the top of your library until you reveal a creature card. Put that card onto the battlefield, then shuffle the rest into your library. If that creature is a Demon, it deals damage equal to its power to each opponent.
+SVar:TrigDig:AB$ DigUntil | Cost$ Mandatory Sac<1/CARDNAME> | Valid$ Creature.YouOwn | ValidDescription$ creature | FoundDestination$ Battlefield | RevealedDestination$ Library | RevealedLibraryPosition$ -1 | RevealRandomOrder$ True | Shuffle$ True | GainControl$ True | SubAbility$ DBDamage | RememberFound$ True
```

Comment:
RevealedLibraryPosition + RevealRandomOrder no longer make sense

GainControl is also redundant for your own zone